### PR TITLE
DDO-3917-scale-down-buffer-pools

### PR DESCRIPTION
**Pull request by jyang-broad**
Looking at the last few weeks of Buffer activity, it looks like we're quite over provisioned on projects, while technically we're not paying for these, it looks like the number of projects is breaking the billing apis via timeout and thus our ability to track costs across the org.

This PR removes what looks like unused pools as well as scales down the big used pools to ~50% of their peak usage.

https://grafana.dsp-devops.broadinstitute.org/d/dcvlwY2nz/resource-buffer-service?orgId=1&var-cluster=terra-integration&var-namespace=terra-tools&var-project=&var-container=All&viewPanel=4&refresh=10s&from=now-30d&to=now

### CODE REVIEW

File: src/main/resources/config/tools/pool_schema.yml [hunk 1]
```diff
@@ -2,30 +2,15 @@
 # READY resource ratio by pool over time: https://grafana.dsp-devops.broadinstitute.org/goto/qHOGMufSk?orgId=1
 ---
 poolConfigs:
-  - poolId: "cwb_ws_fiab_v7"
-    size: 500
-    resourceConfigName: "cwb_ws_resource_fiab_v7"
-  - poolId: "cwb_ws_fiab_v8"
```
Comment (calypsomatic) [inline src/main/resources/config/tools/pool_schema.yml:8]: `cwb_ws_fiab_v8` was just added a few days ago - seems like we might want to keep it around?

File: src/main/resources/config/tools/pool_schema.yml [hunk 2]
```diff
@@ -2,30 +2,15 @@
...
+    size: 1000
     resourceConfigName: "cwb_ws_resource_quality_v11"
   - poolId: "datarepo_v1"
-    size: 2000
+    size: 1000
     resourceConfigName: "datarepo_v1"
-  - poolId: "vpc_sc_qa-fiab_v5"
-    size: 100
-    resourceConfigName: "vpc_sc_qa-fiab_v5"
-  - poolId: "vpc_sc_qa-fiab_v6"
```
Comment (calypsomatic) [inline src/main/resources/config/tools/pool_schema.yml:26]: `vpc_sc_qa-fiab_v6` was also just added a few days ago

File: src/main/resources/config/tools/pool_schema.yml (hunk 1 above)
Comment (pshapiro4broad) [reply src/main/resources/config/tools/pool_schema.yml:8]: Seems likely, the author was @jgainerdewar in PR https://github.com/DataBiosphere/terra-resource-buffer/pull/367

File: src/main/resources/config/tools/pool_schema.yml [hunk 3]
```diff
@@ -2,30 +2,15 @@
...
 poolConfigs:
-  - poolId: "cwb_ws_fiab_v7"
-    size: 500
-    resourceConfigName: "cwb_ws_resource_fiab_v7"
-  - poolId: "cwb_ws_fiab_v8"
-    size: 500
-    resourceConfigName: "cwb_ws_resource_fiab_v8"
-  - poolId: "cwb_ws_quality_v8"
-    size: 5000
-    resourceConfigName: "cwb_ws_resource_quality_v8"
```
Comment (pshapiro4broad) [inline src/main/resources/config/tools/pool_schema.yml:13]: I think each `resourceConfigName` corresponds to a file, should those be deleted too? https://github.com/DataBiosphere/terra-resource-buffer/blob/master/src/main/resources/config/tools/resource-config/cwb_ws_resource_quality_v8.yml

File: src/main/resources/config/tools/pool_schema.yml [hunk 4]
```diff
@@ -2,30 +2,15 @@
...
-    size: 5000
+    size: 1000
     resourceConfigName: "cwb_ws_resource_quality_v10"
   - poolId: "cwb_ws_quality_v11"
-    size: 5000
+    size: 1000
     resourceConfigName: "cwb_ws_resource_quality_v11"
   - poolId: "datarepo_v1"
-    size: 2000
+    size: 1000
```
Comment (pshapiro4broad) [inline src/main/resources/config/tools/pool_schema.yml:12]: This was (somewhat) recently increased from 1500 to 2000. Before changing this to a lower number, we should understand why this was changed in the first place. The PR that changed it had a lot of details: https://github.com/DataBiosphere/terra-resource-buffer/pull/293

Comment (jyang-broad) [reply src/main/resources/config/tools/pool_schema.yml:12]: It was mentioned last time that there was movement on the TDR side to remove/deprecate/update the tests that tend to consume a lot of projects, that and the PR volume is down. 

@snf2ye are you able to comment on if this number seems safe?

File: src/main/resources/config/tools/pool_schema.yml (hunk 2 above)
Comment (jyang-broad) [reply src/main/resources/config/tools/pool_schema.yml]: added this one back

File: src/main/resources/config/tools/pool_schema.yml (hunk 1 above)
Comment (jyang-broad) [reply src/main/resources/config/tools/pool_schema.yml:8]: added this one back, sorry! got fiab v8 confused w/ quality v8 (which _does_ appear to be unused today)

File: src/main/resources/config/tools/pool_schema.yml (hunk 3 above)
Comment (jyang-broad) [reply src/main/resources/config/tools/pool_schema.yml:13]: I went ahead and removed all the config files, checking around to make sure this is a safe-ish move though.

**Comment by jyang-broad** [comment]
hey folks! can I get some thumbs to push this one through?

**Review by pshapiro4broad — APPROVED** [review]
This looks OK.

Is there a way to verify that the schema files are correct, maybe by deploying to a bee?

**Comment by jyang-broad** [comment]
> This looks OK.
> 
> Is there a way to verify that the schema files are correct, maybe by deploying to a bee?

Unfortunately, Buffer never got ported to Bees (we would have had to solve of workspace pooling across multiple buffers as well as buffer orphaned workspaces). I haven't contributed to this repo before, but I expect we can just roll it back

**Review by jgainerdewar — APPROVED** [review]

**Comment by jyang-broad** [comment]
just realized I might have answered a different question than you were asking, (do the new pools work vs will buffer crash) @jgainerdewar mentioned she just tested the new pools in a bee

**Pull request merged**